docs/semgrep-ci/configuration-reference.md

```diff
@@ -21,12 +21,14 @@ SEMGREP_RULES="p/security-audit p/secrets"
 
 ## Suppressing blocking findings or errors
 
-Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep exits with a non-zero exit code (exit code 1) when it reports a blocking finding or there is an issue in running Semgrep (exit code 2 and above). For more information about specific Semgrep exit codes see [CLI reference](../../cli-reference/#exit-codes).
+Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep can be configured to suppress errors (this is the default behavior) or surface all errors depeding on the needs of your specific projects. This behavior is controlled via the `--suppress-errors/--no-suppress-errors` flag, and defaults to suppressing all errors _except_ blocking findings, which surface as exit code 1.
 
-However, Semgrep in CI can run both in fail open and fail closed states. Configure the behavior of Semgrep in CI to suppress blocking findings or internal Semgrep errors blocking your pipeline by using the following options in your YAML configuration file: 
+Semgrep always exits with a non-zero exit code (exit code 1) when it reports a blocking finding. The `--suppress-errors/--no-suppress-errors` controls whether or not semgrep errors in running Semgrep (exit code 2 and above) are surfaced in CI. When the default behavior (equivalent to passing `--suppress-errros`) is left on, Semgrep CI will send an anonymous crash report to a crash-reporting server and then exit with exit code 0. If this behavior is disabled by passing the `--no-suppress-errors` flag to `semgrep ci`, then all exit codes, including internal errors, will be surfaced to the CI provider. For more information about specific Semgrep exit codes see [CLI reference](../../cli-reference/#exit-codes).
```

Comment:
```suggestion
By default, Semgrep blocks your pipeline when it reports a blocking finding (exit code `1`). Blocking findings are defined in [Rule Board](https://semgrep.dev/orgs/-/board) of Semgrep App. Also, the default behavior is such that when Semgrep encounters errors (exit code `2` and above) your pipeline is **not** blocked. You can configure and change the behavior of blocking findings or errors in your CI pipeline by using the following options in your YAML configuration file:
```

Reply:
@adamkvitek if we take this out, then I don't think we have much mention of the use of the `--suppress-errors/--no-suppress-errors` flags on the CLI, right? I'm happy to merge what you wrote with what I wrote, but I want to make sure we preserve that. It also mentions that we send crash reporting to our servers, which I think is important.

Reply:
@spencerdrak we mention it below already, we do not need to double this information (see further suggested edits below).

Reply:
Use the last suggestion from https://github.com/returntocorp/semgrep-docs/pull/651#discussion_r940435686 @spencerdrak